src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -230,7 +416,18 @@ public static EntityBreakBlockEvent createEntityBreakBlock(Game game, Cause caus
         return createEvent(EntityBreakBlockEvent.class, values);
     }
 
-    public static EntityChangeBlockEvent createEntityChangeBlock(Game game, Cause cause, Entity entity, BlockLoc block, BlockSnapshot replacementBlock) {
+    /**
+     * Creates a new {@link EntityChangeBlockEvent}.
+     *
+     * @param game The game instance for this {@link GameEvent}
+     * @param cause The cause of the event, can be null
+     * @param entity The entity involved in this event
+     * @param block The block affected by this event
+     * @param replacementBlock The block that will replace the existing block
+     * @return A new instance of the event
+     */
+    public static EntityChangeBlockEvent
+            createEntityChangeBlock(Game game, Cause cause, Entity entity, BlockLoc block, BlockSnapshot replacementBlock) {
```

Comment:
Hmm, eclipse formatter did that, I'll check for any more occurrences of this and change them.